content/technical_challenges.tex

```diff
@@ -71,36 +71,136 @@ \subsection{Visual Referee Challenge}
 
     \subsubsection{Challenge Goal}
 
-        For the moment robots receive the decision made by the head referee either from a short whistle or a GC message. To improve the perception of the robot and to listen more to the head referee this challenge introduces visual referee signs.
+        In the current SPL rules, the only time that a robot is required to listen directly to the human referees is for the kick-off whistle. Otherwise, all human referee decisions are communicated to the robots via electronic GameController messages. In moving towards the 2050 RoboCup goal, robots will need to directly interpret referee calls and signals (such as whistles, spoken calls and hand signals), rather than receive information from an external electronic source.
+        
+        This technical challenge tests a robot's ability to identify three categories of hand signals:
+        \begin{enumerate}
+            \item Static hand signals with one hand.
+            \item Static hand signals with two hands.
+            \item Dynamic (motion) hand signals with a one or two hands.
+        \end{enumerate}
+        
+        The intent of this challenge is to choose a \emph{subset} of potential referee calls in SPL games and test ability of a robot to recognise different types of hand signals in preparation for adoption in RoboCup games, rather than compile a complete set of all referee signals.
 
     \subsubsection{Challenge setup}
 
-        One robot of the challenged team is placed in the center circle facing the head referee standing on the T-junction opposite to the GC. The referee wears red gloves. The robot watching the referee and listening for the whistle. The general procedure is as follows:
+        One robot of the challenged team is placed in the center circle facing the head referee, standing upright (stiffened) with both hands by its side.
```

Comment:
We should mention how to get the stiff robot from initial into challenge mode. With or without GC.

Reply:
To my understanding GC isn't required for this challenge (if the TCM packages are not being monitored). So I would think in the event of no GC, the robot is simply enabled in the challenge mode by the team when placed. Nothing complicated or explicit needs to be done. If GC is used, then really it's just TCM, and the team can check the robot is connected.